apis/swagger.yml

```diff
@@ -632,7 +632,6 @@ definitions:
       superNodeIp:
          type: "string"
          description: "IP address of supernode that the client can connect to"
```

Comment:
Do we need to update this too?

Reply:
Thanks for remind that.

Reply:
supernode 0.3.x ,cli flag `-Dsupernode.advertiseIp=dragonfly-supernode` is work. since 0.4.x , `--advertise-ip=dragonfly-supernode`  will throws error.

ref https://github.com/dragonflyoss/Dragonfly/blob/master/docs/user_guide/multi_machines_deployment.md#step-1-deploy-dragonfly-server-supernode

> the doc's demo shell show me `--advertise-ip=dfsupernode`
```bash
docker run -d --name supernode --restart=always -p 8001:8001 -p 8002:8002 -v /home/admin/supernode:/home/admin/supernode dragonflyoss/supernode:0.4.2 --download-port=8001 --advertise-ip=dfsupernode
```

Reply:
@anjia0532  It is just an example here and this param only supports ipv4. Actually if supernode is running in Linux, you can remove this param and it also works.